Dn-FamiTracker.rc

```diff
@@ -2020,6 +2029,25 @@ BEGIN
     CONTROL         "Do not remind me again.",IDC_UPDATE_REMIND,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,7,156,213,10
 END
 
+IDD_CONFIG_GUI DIALOGEX 0, 0, 280, 167
+STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION
+CAPTION "GUI"
+FONT 8, "MS Shell Dlg", 400, 0, 0x0
+BEGIN
+    GROUPBOX        "Frame editor",IDC_STATIC,7,6,266,28
+    GROUPBOX        "Register view",IDC_STATIC,7,41,266,42
+    CONTROL         "",IDC_MAXCHANVIEW_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,133,16,91,11
+    LTEXT           "Channel view limit",IDC_STATIC,14,17,58,8
+    LTEXT           "Idle refresh rate",IDC_STATIC,14,52,54,8
+    CONTROL         "",IDC_IDLE_REFRESH_SLIDER,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,133,50,91,12
+    CONTROL         "Smooth register frequency view",IDC_SMOOTHREGFREQ_CHECK,
```

Comment:
I found the new GUI tab in the settings, and thought this would change the pitch register or frequency *text* displays. It actually affects the quantization of the per-channel pitch bar graph.

Ideas:

- Fine pitch {bar/bars/gauge/meter/plot/indicator}
- Fine-grained? Smooth? Precise? Sub-note?

idk.

"Precise pitch bar" sounds half decent.

Reply:
i've given it some thought and wondered if there should be a toggle switch for the text verbosity (i.e. only displaying the registers and channel data, no datatype displayed such as "pitch =" or "size =")